### PR TITLE
Update Tests

### PR DESCRIPTION
**Pull request by yyyaohhh**
#163 #164

### CODE REVIEW

File: src/test/java/seedu/address/logic/parser/ParserUtilTest.java [hunk 1]
```diff
@@ -1,191 +1,197 @@
...
 import org.junit.jupiter.api.Test;
 
+import seedu.address.model.module.Description;
+import seedu.address.model.module.Grade;
+import seedu.address.model.module.ModularCredit;
+import seedu.address.model.module.ModuleCode;
+import seedu.address.model.module.ModuleName;
+import seedu.address.model.module.Semester;
+import seedu.address.model.module.Year;
+import static seedu.address.testutil.Assert.assertThrows;
```
Comment (marquestye) [inline src/test/java/seedu/address/logic/parser/ParserUtilTest.java:14]: Seems to be incorrect import order. Static imports should be above. Can run gradle's checkstyletest to verify.

File: src/test/java/seedu/address/logic/commands/DeleteCommandTest.java [hunk 2]
```diff
@@ -77,4 +135,172 @@ public void toStringMethod() {
...
+        }
+
+        @Override
+        public boolean checkDbValidModuleCode(ModuleCode moduleCode) {
+            return getTypicalModuleData().checkDbValidModuleCode(moduleCode);
+        }
+        @Override
+        public Module getModuleFromDb(ModuleCode moduleCode) {
+            return getTypicalModuleData().getModule(moduleCode);
+        }
```
[the remark this answers is not in the document]
Comment (rocketninja7) [reply src/test/java/seedu/address/logic/commands/DeleteCommandTest.java:251]: I think if it is used then it is correct.

File: src/test/java/seedu/address/logic/commands/DeleteCommandTest.java [hunk 3]
```diff
@@ -77,4 +135,172 @@ public void toStringMethod() {
...
+            throw new AssertionError("This method should not be called.");
+        }
+
+        public Description getDbModuleDescription(ModuleCode moduleCode) throws ModuleNotFoundException {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        public ModularCredit getDbModularCredit(ModuleCode moduleCode) throws ModuleNotFoundException {
+            throw new AssertionError("This method should not be called.");
+        }
```
[the remark this answers is not in the document]
Comment (yyyaohhh) [reply src/test/java/seedu/address/logic/commands/DeleteCommandTest.java:237]: Deleted accordingly👍

File: src/test/java/seedu/address/logic/commands/DeleteCommandTest.java [hunk 4]
```diff
@@ -77,4 +135,172 @@ public void toStringMethod() {
         assertEquals(expected, deleteCommand.toString());
     }
 
+    /**
+     * A default model stub that have all of the methods failing.
+     */
+    private class ModelStub implements Model {
```
[the remark this answers is not in the document]
Comment (yyyaohhh) [reply src/test/java/seedu/address/logic/commands/DeleteCommandTest.java:141]: Created new classes ModelStub

File: src/test/java/seedu/address/logic/parser/ParserUtilTest.java [hunk 5]
```diff
@@ -1,191 +1,197 @@
...
+
+     @Test
+     public void parseCode_validValueWithWhitespace_returnsTrimmedCode() throws Exception {
+         String codeWithWhitespace = WHITESPACE + VALID_CODE + WHITESPACE;
+         ModuleCode expectedCode = new ModuleCode(VALID_CODE);
+         assertEquals(expectedCode, ParserUtil.parseModuleCode(codeWithWhitespace));
+     }
+
+     @Test
+     public void parseMC_null_throwsNullPointerException() {
```
[the remark this answers is not in the document]
Comment (yyyaohhh) [reply src/test/java/seedu/address/logic/parser/ParserUtilTest.java:61]: Checked with gradle checkstyle test and fixed👍

File: src/test/java/seedu/address/logic/commands/DeleteCommandTest.java [hunk 6]
```diff
@@ -77,4 +135,172 @@ public void toStringMethod() {
...
+            throw new AssertionError("This method should not be called.");
+        }
+    }
+
+    /**
+     * A Model stub that contains a single module.
+     */
+    private class ModelStubWithModule extends DeleteCommandTest.ModelStub {
+        private final Module module;
+        final ArrayList<Module> modulesAdded = new ArrayList<>();
```
[the remark this answers is not in the document]
Comment (yyyaohhh) [reply src/test/java/seedu/address/logic/commands/DeleteCommandTest.java:264]: Updated accordingly👍

File: src/test/java/seedu/address/logic/commands/DeleteCommandTest.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (yyyaohhh) [reply src/test/java/seedu/address/logic/commands/DeleteCommandTest.java]: Updated👍

**Pull request merged**